.github/workflows/tag-docker-images.yml

```diff
@@ -18,4 +18,20 @@ jobs:
       - name: Tag and Push
         uses: EventStore/Automations/tag-docker-images@master
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Trigger TestData container building
+        run: |
```

Comment:
There's a bit cleaner way of doing it, maybe want to use that?
https://github.com/orgs/community/discussions/26323#discussioncomment-3251454

Reply:
Indeed I find your approach better

Reply:
It's done now.